src/crypto/PSASessionKeystore.cpp

```diff
@@ -68,12 +68,14 @@ class HkdfKeyAttributes : public KeyAttributesBase
     HkdfKeyAttributes() : KeyAttributesBase(PSA_KEY_TYPE_DERIVE, PSA_ALG_HKDF(PSA_ALG_SHA_256), PSA_KEY_USAGE_DERIVE, 0) {}
 };
 
+#if CHIP_CONFIG_ENABLE_ICD_CIP
```

Comment:
FYI @maciejbaczmanski , following your PR #34925 , it introduced a warning (as error in our build) due to this function being defined but not used, so I put it under the same flag that the method calling it.
I hope this is ok for you.

Reply:
oh, right, I've added this helper later and must've forgotten about the guard... Thanks!